booking/src/main/java/org/dcsa/conformance/standards/booking/BookingComponentFactory.java

```diff
@@ -20,7 +20,10 @@
 
 public class BookingComponentFactory extends AbstractComponentFactory {
   public static final String STANDARD_NAME = "Booking";
-  public static final List<String> STANDARD_VERSIONS = List.of("2.0.0-Beta-1");
+  public static final List<String> STANDARD_VERSIONS = List.of(
+    "2.0.0-beta-1",
+    "2.0.0-beta-2"
```

Comment:
The versions in swagger uses `Beta` with capital `B` (and so do we in eBL as well).

We should be aligned across the board about the casing.